### PR TITLE
✅ test for utils

### PR DESCRIPTION
**Pull request by ssghait007**
- Added unit tests for functions in utils.py

### CODE REVIEW

**Comment by ssghait007** [comment]
@rugglcon changed the PR from draft to ready to review, take a look

File: tests/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+def test_get_background_cmd():
+    t = TestExceptions()
+    t.test_mac_exception()
+    t.test_win_exception()
+    t.test_linux_cmd()
+
+
+def test_change_background():
+    subprocess.call = MagicMock(return_value='Done')
```
Comment (conruggles) [inline tests/test_utils.py]: `subprocess.call` actually returns the return code of the process called, can you mock that value instead? `0` for good and something `> 0` for bad

**Review by conruggles — CHANGES_REQUESTED** [review]
Have one comment on a test.

File: tests/test_utils.py (hunk 1 above)
Comment (ssghait007) [reply tests/test_utils.py]: @rugglcon changed to mock subprocess returncode

**Review by conruggles — APPROVED** [review]

**Comment by conruggles** [comment]
@ssghait007 hey i'm getting an error that it can't find `fake_process` in the `test_utils.py`, where does that come from? I guess I didn't catch that in my review.

**Comment by ssghait007** [comment]
@rugglcon I commented other cases while updating PR for last review fixes.
I created PR before verifying all test cases, let me check this one.

**Comment by conruggles** [comment]
don't worry about it actually, i fixed it myself

**Pull request merged**